packages/bloom/lib/commands/top-k/INFO.ts

```diff
@@ -14,12 +21,43 @@ export default {
     return ['TOPK.INFO', key];
   },
   transformReply: {
-    2: (reply: UnwrapReply<Resp2Reply<TopKInfoReply>>) => ({
-      k: reply[1],
-      width: reply[3],
-      depth: reply[5],
-      decay: Number(reply[7])
-    }),
-    3: undefined as unknown as () => TopKInfoReply
+    2: (reply: UnwrapReply<Resp2Reply<TopKInfoReplyMap>>): TkInfoReply => {
+      return {
+        k: reply[1],
+        width: reply[3],
+        depth: reply[5],
+        decay: Number(reply[7])
```

Comment:
Why just `decay`? what about `width` and `depth`?

Reply:
width/depth are Number's in redis, decay is a double in redis, which in resp2 means a string I believe.  (btw, this isn't a change I made, you did this, this was me trying to intrepret what you did).

see current v5: https://github.com/redis/node-redis/blob/v5/packages/bloom/lib/commands/top-k/INFO.ts / https://github.com/redis/node-redis/blob/fdd1978d92b192c2a85cf50e5dbe5c504270750a/packages/bloom/lib/commands/top-k/INFO.ts

Reply:
why?  a user can force its type mapping to a string....

Reply:
if a user "forces its type to a string" he probably wants a string.. that's a feature, not a bug..

Reply:
ok.  just means differnt types between resp2/3 and was trying to avoid that.

Reply:
ok done.